user_credential_write (#9408)
fishtown-analytics/docs.getdbt.com: website/docs/docs/platform/manage-access/enterprise-permissions.md

- Analyst read is a project-level set.
- Read-only access to project resources, jobs, runs, and environment configs.
- Can access <Constant name="catalog" />.

> - Includes `user_credential_write`, so users can view and edit their own development credentials on **Your profile** > **Credentials** without access to the <Constant name="studio_ide" /> or <Constant name="platform_cli" />.

Reviewer: Can we add some detail around why this is important if the users with this role don't have access to development tools?
```suggestion
- Includes `user_credential_write`, so users can view and edit their own development credentials _for XYZ purposes_...
```
